nomad/client_rpc.go

```diff
@@ -31,6 +31,9 @@ func (s *Server) getNodeConn(nodeID string) (*nodeConnState, bool) {
 	s.nodeConnsLock.RLock()
 	defer s.nodeConnsLock.RUnlock()
 	conns, ok := s.nodeConns[nodeID]
+	if !ok {
```

Comment:
Might be worthwhile to add a test explicitly for this error case

Reply:
https://github.com/hashicorp/nomad/blob/9e761d404998380f002dd7282e9eb665e77b8f74/nomad/client_rpc_test.go#L81-L82

This just reduces nosy logs.